bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdOptionsDefaults.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2023 COSEDA Technologies GmbH and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * Dominic Scharfe (COSEDA Technologies GmbH) - initial implementation
+ *******************************************************************************/
+package org.eclipse.cdt.lsp.clangd;
+
+/**
+ * Interface for providing default options for clangd.
+ */
+public interface ClangdOptionsDefaults extends ClangdOptions {
```

Comment:
Why is this interface needed? Why can't we use `ClangdOptions` ?

Reply:
We discussed this here: https://github.com/eclipse-cdt/cdt-lsp/pull/142#discussion_r1252132102
What I first did was something similar to named bindings in guice, but having a dedicated interface is more clear.